File: benchmark/corpus/11_scoped_budget/b_other_host.py
# Granted one host and port, it sends a request to a different port; the URL comes from input.
import sys
import urllib.request


def ping(url):
    body = urllib.request.urlopen(url).read().decode()  # DANGER
    return "reached: " + body


url = sys.stdin.readline().strip()
print(ping(url))

File: benchmark/corpus/11_scoped_budget/b_other_host.js
// Granted one host and port, it sends a request to a different port; the URL comes from input.
async function ping(url) {
  const body = await (await fetch(url)).text(); // DANGER
  return "reached: " + body;
}

const url = (await new Response(Deno.stdin.readable).text()).trim();
console.log(await ping(url));
